Tests/SourceKitLSPTests/ExecuteCommandTests.swift

```diff
@@ -252,7 +252,278 @@ final class ExecuteCommandTests: XCTestCase {
+              binding.accessorBlock == nil,
+              let type = binding.typeAnnotation?.type
+            else {
+              return []
+            }
+
+            // Ignore the "_storage" variable.
+            if identifier.text == "_storage" {
+              return []
+            }
+
+            guard let defaultValue = binding.initializer?.value else {
+              throw CustomError.message("stored property must have an initializer")
+            }
+
+            return [
+              """
+              get {
+                _storage[\(literal: identifier.text), default: \(defaultValue)] as! \(type)
+              }
+              """,
+              """
+              set {
+                _storage[\(literal: identifier.text)] = newValue
+              }
+              """,
+            ]
+          }
+        }
+
+        @main
+        struct MyMacroPlugin: CompilerPlugin {
+          let providingMacros: [Macro.Type] = [
+            DictionaryStorageMacro.self,
+            DictionaryStoragePropertyMacro.self
+          ]
+        }
+        """#,
+        "MyMacroClient/MyMacroClient.swift": #"""
+        @attached(memberAttribute)
+        @attached(member, names: named(_storage))
+        public macro DictionaryStorage() = #externalMacro(module: "MyMacros", type: "DictionaryStorageMacro")
+
+        @attached(accessor)
+        public macro DictionaryStorageProperty() =
+          #externalMacro(module: "MyMacros", type: "DictionaryStoragePropertyMacro")
+
+        1️⃣@2️⃣DictionaryStorage3️⃣
+        struct Point {
+          var x: Int = 1
+          var y: Int = 2
+        }
+
+        var point = Point()
+        print("Point storage begins as an empty dictionary: \(point)")
+        print("Default value for point.x: \(point.x)")
+        point.y = 17
+        print("Point storage contains only the value we set:  \(point)")
+        """#,
+      ],
+      manifest: SwiftPMTestProject.macroPackageManifest,
+      serverOptions: serverOptions
+    )
+    try await SwiftPMTestProject.build(at: project.scratchDirectory)
+
+    let (uri, positions) = try project.openDocument("MyMacroClient.swift")
+
+    let positionMarkersToBeTested = [
+      (start: "1️⃣", end: "1️⃣"),
+      (start: "2️⃣", end: "2️⃣"),
+      (start: "1️⃣", end: "3️⃣"),
+      (start: "2️⃣", end: "3️⃣"),
+    ]
+
+    for positionMarker in positionMarkersToBeTested {
+      let args = ExpandMacroCommand(
+        positionRange: positions[positionMarker.start]..<positions[positionMarker.end],
+        textDocument: TextDocumentIdentifier(uri)
+      )
+
+      let metadata = SourceKitLSPCommandMetadata(textDocument: TextDocumentIdentifier(uri))
+
+      var command = args.asCommand()
+      command.arguments?.append(metadata.encodeToLSPAny())
+
+      let request = ExecuteCommandRequest(command: command.command, arguments: command.arguments)
+
+      let expectations = [
```

Comment:
You should be able to create an expectation and then set `expectation.expectedFulfillmentCount = 3` instead of an array of expectations.